src/lib/platform/MSWindowsKeyState.cpp

```diff
@@ -1350,6 +1350,17 @@ MSWindowsKeyState::getIDForKey(inputleap::KeyMap::KeyItem& item,
 
 	switch (n) {
 	case -1:
```

Comment:
I think whole case block should be put into braces to silence initialization warning.

Reply:
Fixed this now.